### PR TITLE
allow <BuilderConent name="..." /> to be consistent with BuilderComponent

### PR DESCRIPTION
**Pull request by steve8708**
instead of modelName="..."

### CODE REVIEW

File: packages/react/src/components/builder-content.component.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { VariantsProvider } from './variants-provider.component';
 export interface BuilderContentProps<ContentType> {
   contentLoaded?: (data: any, content: any) => void;
   contentError?: (error: any) => void;
+  model: string;
+  /** @alias for model */
```
Comment (mrkoreye) [inline packages/react/src/components/builder-content.component.tsx:11]: Does this comment make it so typescript knows that `modelName` is not required if you pass `model`, and vice versa? If not, then won't this change make it so bot `model` and `modelName` are required?

Comment (mandx) [reply packages/react/src/components/builder-content.component.tsx:11]: Yeah, this looks like will make both props required.

I believe the way to accomplish that is with something like this:
```typescript
export type BuilderContentProps<ContentType> = {
  // ... the rest of the props
} & ({ model: string } | { modelName: string });
```

Comment (steve8708) [reply packages/react/src/components/builder-content.component.tsx]: doh! great catch, thanks!

File: packages/react/src/components/builder-content.component.tsx [hunk 2]
```diff
@@ -25,6 +24,9 @@ export class BuilderContent<ContentType extends object = any> extends React.Comp
   get builder() {
     return this.props.builder || builder;
   }
+  get name() {
```
Comment (mandx) [inline packages/react/src/components/builder-content.component.tsx:27]:
```suggestion
  get modelName(): string {
```
because explicitness?

**Review by mandx — APPROVED** [review]

**Pull request merged**